### PR TITLE
FOUNDRYUI-80 add markers to range slider

### PR DESCRIPTION
**Pull request by donaldjbrady**
feat(rangeslider): add markers API

### CODE REVIEW

File: packages/hs-react-ui/src/components/RangeSlider/RangeSlider.tsx [hunk 1]
```diff
@@ -156,13 +159,43 @@ export const DomainLabel = styled.div`
...
+
+export const RangeSlider = ({
   StyledContainer = Container,
   StyledDragHandle = DragHandle,
   StyledHandleLabel = HandleLabel,
   StyledSlideRail = SlideRail,
   StyledSelectedRangeRail = SelectedRangeRail,
   StyledDomainLabel = DomainLabel,
+  StyledMarker = Marker,
+  StyledMarkerLabel = MarkerLabel,
```
Comment (joien1) [inline packages/hs-react-ui/src/components/RangeSlider/RangeSlider.tsx:198]: These should get markerProps and markerLabelProps as well

Comment (donaldjbrady) [reply packages/hs-react-ui/src/components/RangeSlider/RangeSlider.tsx:198]: great catch!

**Pull request merged**